sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java

```diff
@@ -134,16 +134,18 @@ public static void setApiUrl(String apiUrl) {
      */
     public static <T> AppCenterFuture<Document<T>> create(String partition, String documentId, T document, Class<T> documentType) {
         AppCenterLog.debug(LOG_TAG, "Create started");
-        getInstance().instanceCreate(partition, documentId, document, documentType);
-        return null;
+        return getInstance().instanceCreateOrUpdate(partition, documentId, document, documentType);
     }
 
     /**
      * Replace a document.
      * The document instance (T) must be JSON serializable.
      */
-    public static <T> AppCenterFuture<Document<T>> replace(String partition, String documentId, T document) {
-        return null;
+    public static <T> AppCenterFuture<Document<T>> replace(String partition, String documentId, T document, Class<T> documentType) {
+        // in the current version we do not have Etag optimistic concurrency logic and the call will do
```

Comment:
Please use `/*` for comments inside methods and always have a blank line before a comment.